server/channels/app/platform/web_conn.go

```diff
@@ -725,7 +725,7 @@ func (wc *WebConn) ShouldSendEvent(msg *model.WebSocketEvent) bool {
 		switch msg.EventType() {
 		case model.WebsocketEventTyping,
 			model.WebsocketEventStatusChange,
-			model.WebsocketEventChannelViewed:
+			model.WebsocketEventMultipleChannelsViewed:
```

Comment:
Just checking, earlier versions of mobile clients don't use the `WebsocketEventChannelViewed` event right?

Reply:
No, so this has to go along with the new version of the mobile client, and the mobile client has to handle both for backwards compatibility. This is the PR that does that: https://github.com/mattermost/mattermost-mobile/pull/7467

Reply:
What would happen to the case where an old mobile client is used with a new server version?

Reply:
The channels would never be marked as read, since now we are using always this new websocket event.

This should not happen often, since the app will be updated automatically from the stores. Only people actively using older versions will suffer from this.

I see this can be a problem for people using V1 and people building their own apps, but I don't know how big of a problem is that. For "normal" users, the app will update directly from the stores, so there is "no possibility" to use an older version.

Reply:
> This should not happen often, since the app will be updated automatically from the stores. 

Atleast in Android, it's not "automatically" upgraded. I don't upgrade my apps unless I want them to. 

No big conerns from my side. I remember that @enahum always wanted to ensure that old mobile clients worked with new server versions as well. As long as it's understood and documented that behavior will break if users don't upgrade their apps, fine by me.

Reply:
I don't think can be avoided in this case, but I'm not worried as apps are updated more regularly than the servers